src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,66 @@
+import { FilterType } from '../../types/Filter';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  numActiveTodos: number;
+  numCompletedTodos: number;
+  statusFilter: FilterType;
+  todos: Todo[];
+  handleClickFilter: (filter: FilterType) => void;
+  handleClearCompleted: () => void;
```

Comment:
props functions should be named as onSmth: onChange, onClick, onClose....

Reply:
It's not really "need" but it "may". So it's just a recommendation. But if you could use one approach through your code it will be great.